src/GHCMod.hs

```diff
@@ -495,12 +181,16 @@ legacyInteractiveLoop symdbreq world = do
     gmPutStr res >> gmPutStrLn "OK" >> liftIO (hFlush stdout)
     legacyInteractiveLoop symdbreq world'
  where
-   interactiveHandlers =
+    interactiveHandlers =
           [ GHandler $ \e@(FatalError _) -> throw e
           , GHandler $ \e@(ExitSuccess) -> throw e
           , GHandler $ \e@(ExitFailure _) -> throw e
           , GHandler $ \(SomeException e) -> gmErrStrLn (show e) >> return ""
           ]
+    locArgs  a (f:l:c:_) = a f (read l) (read c)
```

Comment:
parsing `file`, `line` and `column` from argument list passed to interactive command. This is similar to `locAction` in old argument parser: https://github.com/kazu-yamamoto/ghc-mod/pull/694/files#diff-af20ef78908dc22e249f303d746c2c01L657